reana_server/rest/analyses.py

```diff
@@ -77,24 +79,28 @@ def get_analyses():  # noqa
               [
                 {
                   "id": "256b25f4-4cfb-4684-b7a8-73872ef455a1",
+                  "name": "name_of_workflow",
```

Comment:
Although I guess that you probably discarded using `workflow_name` because it is the name of the parameter itself, this name sounds weird to me, what about using the same as in [`reana-client`](http://reana-client.readthedocs.io/en/latest/gettingstarted.html#create-workflow) `mytest` for one and `mytest-n` for lists?

Reply:
Yes WIP